### PR TITLE
feat(types,clerk-js): Introduce web3 wallet operations in UserProfile

### PR DESCRIPTION
**Pull request by chanioxaris**
## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [x] `@clerk/types`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend-core`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/edge`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

Allow users to list, preview, create/connect and delete a web3 wallet from their UserProfile, the same way they do for email addresses & phone numbers

Screenshot 2022-05-18 at 12 32 15 PM

Screenshot 2022-05-18 at 12 32 10 PM

Screenshot 2022-05-18 at 12 31 59 PM

### CODE REVIEW

**Review by nikosdouvlis — APPROVED** [review]
This looks good to me!

Let's wait for a second review as I'm not entirely familiar with all web3 flows :)

File: packages/clerk-js/src/ui/userProfile/account/profileCard/ProfileCard.tsx
Comment (SokratisVidros) [inline packages/clerk-js/src/ui/userProfile/account/profileCard/ProfileCard.tsx:103]: 💯

**Review by SokratisVidros — APPROVED** [review]
Good job @chanioxaris! 👏

File: packages/clerk-js/src/ui/userProfile/account/profileCard/ProfileWeb3Wallets.tsx
Comment (igneel64) [inline packages/clerk-js/src/ui/userProfile/account/profileCard/ProfileWeb3Wallets.tsx:18]: 🙃 
I would prefer a ternary here based on `user.web3Wallets.length`

File: packages/clerk-js/src/ui/userProfile/web3Wallets/AddNewWeb3Wallet.tsx
Comment (igneel64) [inline packages/clerk-js/src/ui/userProfile/web3Wallets/AddNewWeb3Wallet.tsx:68]: 🙃 
Let's kill `==` vs `===` in eslint please.

Comment (nikosdouvlis) [reply packages/clerk-js/src/ui/userProfile/web3Wallets/AddNewWeb3Wallet.tsx:68]: Nice catch, totally missed that

**Review by igneel64 — APPROVED** [review]

**Comment by chanioxaris** [comment]
All comments have been addressed

**Pull request merged**
